2.hello-kubernetes/python/app.py

```diff
@@ -0,0 +1,23 @@
+# ------------------------------------------------------------
```

Comment:
Remove this file since it has been a decison in the past to merge the sample code from samples 1 and 2.

Reply:
@artursouza I know we've gone back and forth on this, but I think it's confusing for people to open sample two and not see the source. Even if it's the same code, I think we should consider having it in both places.